### ISSUE TITLE
Progress information for doc to PDF conversion

### ISSUE THREAD

**Issue by deeplow** [freedomofpress/dangerzone#432]
We currently don't have a way to see progress information when converting a libreoffice document to a PDF.

However, we may be able to to use libreoffice's [page-range](https://github.com/freedomofpress/dangerzone/issues/424#issuecomment-1567021048) option to do it in batches of N pages.

**Comment by deeplow** [comment]
@apyrgio stated in a verbal discussion that this will probably introduce overhead since we are starting libreoffice multiple times.

A solution to that could be to use the [unoserver](https://github.com/unoconv/unoserver/), a tool that's made to solve this exact problem.

**Comment by apyrgio** [comment]
Nice find, and it makes sense to have it in mind for the #424. For the Qubes integration though, would it make sense to follow an iterative process, where we first tally the time LibreOffice takes to finish the conversion into the time it takes to separate the pages? Not accurate, I know, but it's a first step towards a working progress counter.

**Comment by deeplow** [comment]
> For the Qubes integration though, would it make sense to follow an iterative process, where we first tally the time LibreOffice takes to finish the conversion into the time it takes to separate the pages? Not accurate, I know, but it's a first step towards a working progress counter.

Yes, we can start with what we already have (fixed % for this step of the process) and later change to protocol to accommodate more granular progress information.
